content/10-managing-databases/07-introduction-database-caching.mdx

```diff
@@ -0,0 +1,97 @@
+For example, your application requests user information from the database for the first time, this request goes from server to server and returns back the requested information. With caching, this user profile is stored closer to the requester after initial read, and there is a significant reduction in query processing time and database workload for all subsequent read requests for that data.
+
+## What are the benefits of database caching?
+
+Data retrieval speed greatly affects the user experience of an application. Implementing a caching strategy on your database can result in improved database performance, availability, and scalability for minimal cost depending on the strategy, all factors contributing to an overall positive application experience.
+
+### Performance
+As touched on briefly, database caching improves the performance of a database by making data more easily accessed. The cache acts has a sort of “keyboard short-cut” or “hot-key” for the application to reference data that it frequently is calling upon. 
+
+This speedier request can minimize the workload of the database, keeping it from spending inefficient amounts of time doing repetitive tasks. Instead making these tasks more efficient and simplifying data access. 
+
+### Availability
+While not a 100% failover strategy, caching also provides benefits to overall database availability. Depending on where the cache is stored, the cache can still provide a place for the application to call upon for data in the case of the primary database server becoming unavailable for any reason.
+
+While database performance is generally the primary reason for adopting a caching strategy, you also have the added benefit of some additional resiliency in the case of any backend failures. 
+
+### Scalability
+Similarly to added high availability, database caching has a positive effect on scalability. While it shouldn’t be your main consideration for a database scaling strategy, implementing caching to improve database performance reduces your database workload, therefore distributing backend queries across entities. 
+
+This distribution lightens the load on a primary database and can reduce costs and provide more flexibility in the processing of your data. This results in a more scalable environment as you continue for take more frequently requested data off the plate of the primary database.
+
+## What are the different database caching strategies?
+
+Before adopting database caching into your data access flow, it is important to consider which caching strategy is best suited for the job. For any scenario, the relationship between the database and the cache can have a different impact on performance and system structure. Planning ahead and considering all options will lead to less headaches down the road. 
+
+The four most popular strategies to consider are cache aside, read through, write through, and write back. We’ll cover the data source to cache relationship and process of each strategy.
```

Comment:
Really like this section description. :+1:

Reply:
Just noticed, you might want to add "write around" to the list of strategies you'll be covering.

Reply:
👍 I think I initially thought to skip it because it doesn't offer much differentiation. Will add it :)